cdap-ui/app/features/workflows/controllers/tabs/runs/runs-detail-ctrl.js

```diff
@@ -3,7 +3,9 @@ class WorkflowsRunsDetailController {
     let filterFilter = $filter('filter');
     let match = filterFilter($scope.RunsController.runs, {runid: $state.params.runid});
     $scope.RunsController.runs.selected.runid = match[0].runid;
-
+    $scope.RunsController.runs.selected.status = match[0].status;
+    $scope.RunsController.runs.selected.start = match[0].start;
+    $scope.RunsController.runs.selected.end = match[0].end;
```

Comment:
Talked to @elfenheart. We don't want to overwrite the object reference here since it is also being used in the parent controller (RunsController)